applications/daily_tests/automatic_recovery_test.js

```diff
@@ -84,14 +84,14 @@ async function run(options = {}) {
         let successLog = data.match(RECOVERY_COMPLETE_REGEXP);
         let recoveredAmount = data.match(RECOVERY_WORTH_REGEXP);
         if (successLog && recoveredAmount) {
-          let recoveredAmount = parseInt(recoveredAmount[1]);
+          let recoveredAmt = parseInt(recoveredAmount[1]);
```

Comment:
nit: I prefer the full typed out name, it's more readable than Amt.

Reply:
It's already used - you cannot rebind variables in js. I will change the variable names to be better

Reply:
`let` is block scoped so you should be able to reuse the name if you really want to 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let

Reply:
yeah, but that var name is in this scope :P

Reply:
Changed it in any case to `recoveredAmountMatch` and `recoveredAmount`